### PR TITLE
Added missing links

### PR DESCRIPTION
**Pull request by Beta-King**
#123 I've solved this issue :D

### CODE REVIEW

**Comment by Beta-King** [comment]
https://beta-king.github.io/Dream2018.fossasia.org/ --> preview link !!

**Comment by Beta-King** [comment]
please review; if there are no conflicts; can we merge ??

**Comment by Beta-King** [comment]
??

**Comment by Beta-King** [comment]
Please review :D

**Comment by Beta-King** [comment]
can we merge ??

**Comment by Beta-King** [comment]
@mariobehling could you merge it please :D

**Comment by Beta-King** [comment]
??

File: index.html [hunk 1]
```diff
@@ -93,6 +93,7 @@
                                     <li class="social-link"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
                                     <li class="social-link"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
                                     <li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
+				    <li class="social-link"><a target="_self" href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a></li>
```
Comment (abishekvashok) [inline index.html]: Add a single space to intend this line

Comment (Beta-King) [reply index.html]: it is already indented; its just showing like that - wait ill send an image :D

Comment (Beta-King) [reply index.html]: screen shot 2018-01-02 at 9 50 25 am

Comment (Beta-King) [reply index.html]: Look the line 96 where i have made the changes; it is indented properly :D

**Comment by Beta-King** [comment]
@mariobehling, Hey could you please review and merge this pull request :D

**Comment by Beta-King** [comment]
@mariobehling please review and merge this :D

**Comment by Beta-King** [comment]
Mentioned the issue :D

**Comment by Beta-King** [comment]
please merge guys :D

**Comment by Beta-King** [comment]
@mariobehling, please merge it :D

**Comment by mariobehling** [comment]
Thanks.

**Comment by Beta-King** [comment]
guys can we merge ?? :D

**Comment by Beta-King** [comment]
oh, ohk thanks :D

**Pull request merged**